docs/reference/security/authentication/saml-guide.asciidoc

```diff
@@ -328,24 +328,18 @@ groups:: _(Recommended)_
 +
 [NOTE]
 ====
-Some IdPs are configured to send the `groups` list as a comma-separated string,
-but {es} can't parse this string into an array of groups. To map this SAML
-attribute to the `attributes.groups` setting in the {es} realm, a cluster
-security administrator can use a wildcard when
-<<saml-role-mapping,configuring role mappings>>. While flexible, wildcards are
```

Comment:
I'm wondering if we should keep the part about wildcard matching around for when `attribute_delimiters.group` doesn't cut it. I'd also avoid the duplication of the setting description. The docs flow could be something like:

Some IdPs are configured to send the `groups` list as a comma-separated string. {es} supports splitting this string into an array of groups. Use the `attribute_delimiters.group` setting to specify a delimiter to split on.

If your IdP is configured to send the `groups` list in format that cannot be parsed with simple delimiter-based splitting, you can use a wildcard when <<saml-role-mapping,configuring role mappings>>, to map this SAML
attribute to the `attributes.groups` setting in the {es} realm. While flexible, ....

LMKWYT!

Reply:
> I'm wondering if we should keep the part about wildcard matching around for when attribute_delimiters.group doesn't cut it.

I actually left that out on purpose because I feel like the point of the note is to address the case when you have a single csv string as the group value. My understanding from the [issue](https://github.com/elastic/elasticsearch/issues/84379) is that the mapping + regexp would entirely be replaced by this new delimiter approach. Based on this it sounds like it's pretty rare, and when it occurs it's only with csv: 

> Do you have a feeling on how common the single-value scenario is, or if it affects common IdP?
I don't know of this to be a limitation of any well-known (or even just commonly used) IdP implementations. I think this is mostly a configuration issue on the IdP administration side, rather than a software limitation. I agree with the arguments around doing something about this, nevertheless.

> I'd also avoid the duplication of the setting description

Yes, I agree.

I've updated the docs. Let me know what you think!

Reply:
> Based on this it sounds like it's pretty rare, and when it occurs it's only with csv

Yup that's fair. I'm good with your approach -- if we run into issues where it doesn't suffice we can revive part of the old blurb. As you say this should be rare in the first place, and I don't have evidence to claim that an IdP is sending a delimiter separated value that is _not_ covered by the new setting (e.g., multi-value).